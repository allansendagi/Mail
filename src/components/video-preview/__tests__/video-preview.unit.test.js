import React from 'react';
import {shallow, configure} from 'enzyme';
import {VideoPreview} from '../video-preview.component';
import Adapter from 'enzyme-adapter-react-16';


configure({adapter: new Adapter()});
describe('VideoPreview', () => {
  test('renders vertically', () => {
    const wrapper = shallow(<VideoPreview/>);
    expect(wrapper).toMatchSnapshot();
  });
  test('renders horizontally', () => {
    const wrapper = shallow(<VideoPreview horizontal={true}/>);
    expect(wrapper).toMatchSnapshot();
  });
});