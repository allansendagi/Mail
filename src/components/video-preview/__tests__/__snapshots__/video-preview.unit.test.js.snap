// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`VideoPreview renders horizontally 1`] = `ShallowWrapper {}`;

exports[`VideoPreview renders vertically 1`] = `ShallowWrapper {}`;
